src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -1097,5 +1097,13 @@ void GcsActorManager::KillActor(const std::shared_ptr<GcsActor> &actor) {
   RAY_UNUSED(actor_client->KillActor(request, nullptr));
 }
 
+void GcsActorManager::AddDestroyedActorToCache(const std::shared_ptr<GcsActor> &actor) {
+  if (destroyed_actors_.size() >=
+      RayConfig::instance().maximum_gcs_destroyed_actor_cached_count()) {
+    destroyed_actors_.erase(destroyed_actors_.begin());
```

Comment:
Are we repulling this actor if they are not in cache? Was it implemented in a separate PR?

Reply:
How about do not  re-pull actors if they are not in cache？Destroyed actors only show maximum_gcs_destroyed_actor_cached_count.